client/src/components/profile/memberscard.js

```diff
@@ -0,0 +1,140 @@
+import withStyles from '../jss'
+import { getMembers, addMember, removeMember } from '../../api/members'
+import { updateAccount } from '../../api/profile'
+import { useState, useCallback, useEffect } from 'preact/hooks'
+import config from '../../../../config/client'
+
+import Form from '../form'
+import { useToast } from '../toast'
+
+const MemberRow = withStyles({
+  root: {
+    alignItems: 'center',
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'space-between'
+  }
+}, ({ classes, id, name, setMembers }) => {
+  const { toast } = useToast()
+
+  const handleDelete = useCallback(() => {
+    removeMember({ id })
+      .then(() => {
+        setMembers(members => members.filter(a => a.id !== id))
+
+        toast({ body: 'Team member successfully deleted' })
+      })
+  }, [id, setMembers])
+
+  return (
+    <div class={classes.root} key={id}>
+      <p class='u-no-margin'>{name}</p>
+      <div class='btn-container u-vertical-center'>
+        <input onClick={handleDelete} type='submit' class='btn-tiny btn-danger u-no-margin' value='Delete' />
+      </div>
+    </div>
+  )
+})
+
+const MembersCard = withStyles({
+
```

Comment:
We can remove this `withStyles` HOC.

Reply:
I think it's fine to leave in, to make it easier to style in future.